conf: updated build.gradle to add CF local dependency.

aagarwal1012 opened the pull request:
@wmdietl, please review.

kelloggm commented on build.gradle:
```diff
@@ -13,7 +13,11 @@ sourceCompatibility = 1.8
+    
+    compile files('${CHECKERFRAMEWORK}/checker/dist/checker-qual.jar')
+    compile files('${CHECKERFRAMEWORK}/checker/dist/checker.jar')
+    compile files('${CHECKERFRAMEWORK}/checker/dist/jdk8.jar')
```
The `compile` dependency configuration is considered deprecated (https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_plugin_and_dependency_management); you should use `implementation` instead.

wmdietl commented:
The maven configuration of guava uses a flag to use a local Checker Framework:
https://github.com/typetools/guava/blob/master/guava/pom.xml#L72

I would hope something similar is easy to do with gradle.

aagarwal1012 commented:
@kelloggm, please check the updated `build.gradle` file.
* Added variable `checkerframework_local` to add local dependencies of CF.
* Changed `compile` to `implementation`.
